train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java

```diff
@@ -48,7 +48,8 @@ private void enforceSpeedLimit() {
 
 	@Override
 	public void setJoystickPosition(int joystickPosition) {
-		this.step = joystickPosition;		
+		this.step = joystickPosition * 2;
```

Comment:
Why is 2? The joystick position should change the reference speed of the Train, in concordance with the specification found in the Readme.md